ci/function_images_check.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+echo "Running function image check..."
+
+images=($(cat functions.json | jq '.functions[].images[]' | sed 's/"//g;s/:/\/tags\//g'))
+
+for image in "${images[@]}"; do 
+
+    url="https://hub.docker.com/v2/repositories/${image}/"
+    http_status=$(curl -Iks -X GET $url 2>/dev/null | head -n 1 | cut -d' ' -f2)
+
+    if ! [ "$http_status" == "200" ]; then
+        echo "Failure: cURL of address $url returned a non-200 status code, exiting..."
```

Comment:
It might be nice to have the `$image` in the error message as well as the `$url`, it will standout a little better than the url

Reply:
I've split the manipulation into two parts now so we enter the loop with an image name and then build the stem within the loop.